src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -90,23 +98,30 @@ class psElevateTokenHandler {
     * Rejects any messages from an unknown origin.
     */
     registerPostMessageListener(component) {
-        let self = this;
-
+        const self = this;
         window.onmessage = async function (event) {
-
-            if (self._visualforceOriginUrls) {
-                self._visualforceOrigin = self._visualforceOriginUrls.find(
-                    origin => event.origin === origin.value
-                ).value;
-            }
-
-            if (self._visualforceOrigin) {
+              if (self.shouldHandleMessage(event)) {
                 const message = JSON.parse(event.data);
                 component.handleMessage(message);
             }
         }
     }
 
+    shouldHandleMessage (event) {
```

Comment:
Let's add descriptions to new methods providing more information about methods.

Reply:
Updated!